core/render/json/default.py

```diff
@@ -138,10 +138,9 @@ def renderSingleBoneValue(self, value, bone, skel, key):
 		if bone.type == "date" or bone.type.startswith("date."):
 			if value:
 				if bone.date and bone.time:
-					return value.strftime("%d.%m.%Y %H:%M:%S")
+					return value.strftime("%d.%m.%Y %H:%M:%S%z")
```

Comment:
Wouldn't it be easier and sufficient to simply use datetime.isoformat(' '), which is equivalent to __str__, for all cases, i.e.:
if value:
  return str(value)